src/openzeppelin/tests/mocks/non721_mock.cairo

```diff
@@ -0,0 +1,7 @@
+#[contract]
+mod NonERC721Mock {
+    #[view]
+    fn nope() -> bool {
```

Comment:
Since other dual contracts will have similar tests, what do you think about renaming this to something more generic so it can be reused? E.g. `incompatible_mock`

Reply:
`nope()`